### PR TITLE
fix: use flashbots builder

### PR DESCRIPTION
**Pull request by h4ck3rk3y**

### CODE REVIEW

File: src/package_io/parse_input.star [hunk 1]
```diff
@@ -255,16 +255,15 @@ def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port):
...
 		# TODO(maybe) make parts of this more passable like the mev-relay-endpoint & forks
-		"el_extra_params": ["--builder",  "--builder.remote_relay_endpoint=http://mev-relay-api:9062", "--builder.beacon_endpoints=http://cl-{0}-lighthouse-geth:4000".format(num_participants+1), "--builder.bellatrix_fork_version=0x30000038", "--builder.genesis_fork_version=0x10000038", "--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER),  "--miner.extradata=\"Illuminate Dmocratize Dstribute\"", "--miner.algotype=greedy"] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
+		"el_extra_params": ["--builder",  "--builder.remote_relay_endpoint=http://mev-relay-api:9062", "--builder.beacon_endpoints=http://cl-{0}-lighthouse-geth:4000".format(num_participants+1), "--builder.bellatrix_fork_version=0x30000038", "--builder.genesis_fork_version=0x10000038", "--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER),  "--miner.extradata=\"Illuminate Dmocratize Dstribute\"", "--builder.algotype=greedy"] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
```
Comment (barnabasbusa) [inline src/package_io/parse_input.star:266]: its pretty hard to read individual params, could you make this a multi line instead?

**Review by barnabasbusa — APPROVED** [review]

**Comment by h4ck3rk3y** [comment]
Blocked on - https://github.com/flashbots/builder/issues/107

**Comment by barnabasbusa** [comment]
Pls change to `ethpandaops/flashbots-builder:main`

**Comment by h4ck3rk3y** [comment]
@barnabasbusa done! i am just running this locally agian

**Pull request merged**